src/imp/tracer_imp.js

```diff
@@ -883,11 +883,13 @@ export default class TracerImp extends EventEmitter {
         let clockOffsetMicros = this._clockState.offsetMicros();
 
         // Diagnostic information on the clock correction
-        this.logStable('cr/time_correction_state', {
-            offset_micros  : clockOffsetMicros,
-            active_samples : this._clockState.activeSampleCount(),
-            ready          : clockReady,
-        });
+        if (this._options.debug) {
```

Comment:
As far as I understand, mocha is not supposed to run tests in parallel. If it seems to be doing that, then I suspect something else is going wrong.